cmd/deploytarget.go

```diff
@@ -417,20 +421,24 @@ var RemoveDeployTargetFromOrganizationCmd = &cobra.Command{
 			debug)
 
 		organization, err := l.GetOrganizationByName(context.TODO(), organizationName, lc)
-		handleError(err)
+		if err := handleErr(err); err != nil {
```

Comment:
These just needs to be like this, there isn't a reason to use `handleErr`.
```
if err != nil {
	return err
}
```

When a command with `RunE` errors, it prints the help information in the same way that `handleErr` does

Reply:
`handleErr` doesn't print the help information, if the err is caught it returns `nil` only the error message is output.

Reply:
Yeah, I don't think that wrapper is even useful here, the results are this:
```
$ go run main.go add organization-deploytarget --name fakeorg --deploytarget 123
Error: graphql: Organization doesn't exist
```

vs just simply doing this, and letting cobra handle the error
```
if err != nil {
	return err
}
```

gets this, with the error and full help text.
```
$ go run main.go add organization-deploytarget --name fakeorg --deploytarget 123
Error: graphql: Organization doesn't exist
Usage:
  lagoon add organization-deploytarget [flags]

Aliases:
  organization-deploytarget, org-dt

Flags:
  -D, --deploytarget uint   ID of DeployTarget
  -h, --help                help for organization-deploytarget
  -O, --name string         Name of Organization

Global Flags:
      --config-file string   Path to the config file to use (must be *.yml or *.yaml)
      --debug                Enable debugging output (if supported)
  -e, --environment string   Specify an environment to use
      --force                Force yes on prompts (if supported)
  -l, --lagoon string        The Lagoon instance to interact with
      --no-header            No header on table (if supported)
      --output-csv           Output as CSV (if supported)
      --output-json          Output as JSON (if supported)
      --pretty               Make JSON pretty (if supported)
  -p, --project string       Specify a project to use
      --skip-update-check    Skip checking for updates
  -i, --ssh-key string       Specify path to a specific SSH key to use for lagoon authentication

Error: graphql: Organization doesn't exist
exit status 1
```

While the wrapper does omit all the other stuff, it makes the error handling a bit confusing

Reply:
Ah yep, if you're ok with the full help text displaying on api/graph errors then it makes sense to change it. I'll update it now

Reply:
Yeah, its just much simpler error handling logic, and every other command that uses `RunE` does this. It is only the older commands that don't.